test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap

```diff
@@ -2284,19 +2284,19 @@ svg.jhxt971g {
 }
 ,
   "externals.json": "[
-  "crypto",
-  "fs",
-  "os",
-  "path",
-  "stream",
-  "tty",
-  "url",
-  "util"
+  "external node-commonjs \\"crypto",
+  "external node-commonjs \\"fs",
+  "external node-commonjs \\"os",
+  "external node-commonjs \\"path",
+  "external node-commonjs \\"stream",
+  "external node-commonjs \\"tty",
+  "external node-commonjs \\"url",
+  "external node-commonjs \\"util"
```

Comment:
Might be nothing, but these look like unterminated strings.

Reply:
Found the culprit https://github.com/seek-oss/sku/blob/master/test/utils/ListExternalsWebpackPlugin.js.
I think something has changed, will try tweak it.